Dockerfile

```diff
@@ -20,7 +23,7 @@ RUN curl -o ~/miniconda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest
      ~/miniconda.sh -b -p /opt/conda && \
      rm ~/miniconda.sh && \
      /opt/conda/bin/conda install -y python=$PYTHON_VERSION numpy pyyaml scipy ipython mkl mkl-include && \
-     /opt/conda/bin/conda install -y pytorch $PYTORCH_DEPS -c pytorch && \
+     /opt/conda/bin/conda install -y pytorch=1.11 $PYTORCH_DEPS -c pytorch && \
```

Comment:
this choice was deliberate. Before Gitlab restricted their CI, the docker image represented the minimal config needed to use SB3 (so python 3.7 and minimum torch version), but yes, it would make sense to have an env variable or remove the pinned version (I must have added it to force the correct pytorch version).
Switching to cog would probably solve that issue too.